x-pack/plugins/security_solution/public/timelines/components/timeline/data_providers/empty.tsx

```diff
@@ -73,38 +74,42 @@ const NoWrap = styled.div`
 NoWrap.displayName = 'NoWrap';
 interface Props {
   showSmallMsg?: boolean;
+  timelineId: string;
 }
 /**
  * Prompts the user to drop anything with a facet count into the data providers section.
  */
-export const Empty = React.memo<Props>(({ showSmallMsg = false }) => (
+export const Empty = React.memo<Props>(({ showSmallMsg = false, timelineId }) => (
   <EmptyContainer
     className="timeline-drop-area-empty"
     data-test-subj="empty"
     showSmallMsg={showSmallMsg}
   >
     {!showSmallMsg && (
       <>
-        <NoWrap>
-          <Text color="subdued" size="s">
-            {i18n.DROP_ANYTHING}
-          </Text>
-          <HighlightedBackground>
-            <BadgeHighlighted className={HIGHLIGHTED_DROP_TARGET_CLASS_NAME}>
-              {i18n.HIGHLIGHTED}
-            </BadgeHighlighted>
-          </HighlightedBackground>
-        </NoWrap>
-
-        <NoWrap>
-          <Text color="subdued" size="s">
-            {i18n.HERE_TO_BUILD_AN}
-          </Text>
-          <AndOrBadge type="or" />
-          <Text color="subdued" size="s">
-            {i18n.QUERY}
-          </Text>
-        </NoWrap>
+        <div>
+          <NoWrap>
+            <Text color="subdued" size="s">
+              {i18n.DROP_ANYTHING}
+            </Text>
+            <HighlightedBackground>
+              <BadgeHighlighted className={HIGHLIGHTED_DROP_TARGET_CLASS_NAME}>
+                {i18n.HIGHLIGHTED}
+              </BadgeHighlighted>
+            </HighlightedBackground>
+            <Text color="subdued" size="s">
+              {i18n.HERE_TO_BUILD_AN}
+            </Text>
+            <AndOrBadge type="or" />
+            <Text color="subdued" size="s">
+              {i18n.QUERY}
+            </Text>
+          </NoWrap>
+        </div>
+
+        <div>
```

Comment:
I am worried that `AddDataProviderPopover` is returning a EuiFlexItem but it is not contain inside of `EuiFlexGroup`